SPARCoreTests/SPARCoreTests.m

```diff
@@ -1,12 +1,8 @@
-//
-//  SPARCoreTests.m
-//  SPARCoreTests
-//
-//  Created by Alex Mitchell on 2/11/17.
-//  Copyright © 2017 comp. All rights reserved.
-//
+
 
 #import <XCTest/XCTest.h>
+#import "GADArticle.h"
```

Comment:
Add a newline between the framework imports (in angle brackets) and project imports (in quotations):

```
#import <XCText/XCTest.h>

#import "GADArticle.h"
```